docusaurus.config.js

```diff
@@ -49,7 +49,9 @@ module.exports = {
       darkTheme: require("prism-react-renderer/themes/dracula"),
       additionalLanguages: ["java", "ruby", "php"],
     },
-    // hideableSidebar: true,
+    sidebar: {
+      autoCollapseCategories: true,
+    },
```

Comment:
```suggestion
    docs: {
        sidebar: {
            hideable: true,
            autoCollapseCategories: true,
        },
    },
```
Per:
https://github.com/facebook/docusaurus/blob/main/website/docusaurus.config.js#L363

Seems like it is more like this?